tests/test-travis/test-tlmgr.R

```diff
@@ -17,6 +17,12 @@ assert('`tlmgr info` can list the installed packages', {
   (c('xetex', 'luatex', 'graphics') %in% res)
 })
 
+assert('`tl_size()` can correctly list name and size', {
+  res = tl_sizes(pkgs = "luatex")$package
+  # only check a few basic packages
+  ('luatex' %==% tl_sizes(pkgs = "luatex")$package)
```

Comment:
@cderv Out of curiosity, where is the `%==%` operator from?

GitHub search was not very useful.

Reply:
It is from **testit** framework that we are using for testing in this package
https://github.com/yihui/testit